cfgov/cfgov/settings/base.py

```diff
@@ -123,9 +125,13 @@
     {
         'NAME': 'wagtail-env',
         'BACKEND': 'django.template.backends.jinja2.Jinja2',
-        'DIRS': [V1_TEMPLATE_ROOT, V1_TEMPLATE_ROOT.child('_includes'),
+        'DIRS': [
+            V1_TEMPLATE_ROOT,
+            V1_TEMPLATE_ROOT.child('_includes'),
             V1_TEMPLATE_ROOT.child('_layouts'),
-            PROJECT_ROOT.child('static_built')],
+            DATA_RESEARCH_TEMPLATE_ROOT,
```

Comment:
> Centralizing also reduces possibility for name conflict 

That was an immediate concern that I thought of when doing this, I thought I'd just add this in controversially ;)

I think I lean towards a central location as well, but it should probably be reworked. For example, let's remove the `v1` folder and just have everything live directly under jinja2. This type of change is out of scope for this PR though. I can move it back to status quo if requested.